src/Library/demos/Toolbar View/main.blp

```diff
@@ -0,0 +1,220 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.Window {
+  width-request: 360;
+  height-request: 200;
+  default-width: 640;
+  default-height: 640;
+
+  Adw.ToolbarView toolbar_view {
+    vexpand:true;
+    extend-content-to-bottom-edge: true;
+    extend-content-to-top-edge: true;
+    top-bar-style: bind barstyle_select.selected;
+    bottom-bar-style: bind barstyle_select.selected;
+    reveal-top-bars: false;
+    reveal-bottom-bars: false;
+    [top]
+    Adw.HeaderBar header_top {
+      title-widget: Label{
+      label:_("Header Bar");
+      };
+    }
+    content: Adw.StatusPage {
```

Comment:
Switch this to an `Adw.ViewStack`

Reply:
Why? there is nothing to switch to

Reply:
Sorry, I'm confused now. I thought you wanted to demonstrate all the different bars? Unless I'm mistaken you need an `Adw.ViewStack` for the `Adw.ViewSwitcherBar` to work?

Reply:
Ah, I guess the idea is to have that other view switcher in the `.blp` file, along with the other widget? I guess I'm confused why not just actually use a view switcher?

Reply:
[Kooha-2023-08-17-18-11-54.webm](https://github.com/sonnyp/Workbench/assets/96797205/b7de120e-6368-455a-bebc-8c99ff57e980)

@sonnyp @andyholmes  Is this what we wanted to achieve?